docs/src/basics/solve.md

```diff
@@ -3,7 +3,3 @@
 ```@docs
 solve(prob::IntegralProblem, alg::SciMLBase.AbstractIntegralAlgorithm)
 ```
-
-Additionally, the extra keyword arguments are splatted to the library calls, so
-see the documentation of the integrator library for all the extra details.
-These extra keyword arguments are not guaranteed to act uniformly.
```

Comment:
This is a standard across all other types because of the interaction with ensembles. Is there a reason to drop it here?

Reply:
It seems this is covered i nhttps://github.com/SciML/Integrals.jl/pull/244 ?